Sources/Commands/SwiftTool.swift

```diff
@@ -313,9 +313,16 @@ public class SwiftTool<Options: ToolOptions> {
         if let workspace = _workspace {
             return workspace
         }
+
         let delegate = ToolWorkspaceDelegate()
         let rootPackage = try getPackageRoot()
-        let provider = GitRepositoryProvider(processSet: processSet)
+
+        // Create git provider.
+        let gitOverrides = rootPackage.appending(component: "git-overrides.json")
```

Comment:
Do we really want this file to come from the root package path versus, say, an environment variable?